API/models/model.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../../db/connection");
+const format = require("pg-format");
+
+exports.fetchTable = (table) => {
+  const queryString = format("SELECT * FROM %I", table);
+  return db.query(queryString).then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "Table is Empty" });
```

Comment:
Why would it be an error if a table has no rows in it? I think you're overthinking this end point, this should be a select * from topics which when looking at your test and dev data, there will be some.